video/out/opengl/video.c

```diff
@@ -3391,7 +3393,8 @@ static void check_gl_features(struct gl_video *p)
     bool have_mglsl = gl->glsl_version >= 130; // modern GLSL (1st class arrays etc.)
     bool have_texrg = gl->mpgl_caps & MPGL_CAP_TEX_RG;
     bool have_tex16 = !gl->es || (gl->mpgl_caps & MPGL_CAP_EXT16);
-    bool have_compute = gl->glsl_version >= 430; // easiest way to ensure all
+    bool have_compute = gl->DispatchCompute;
```

Comment:
Actually, my point is, even if we want to combine these three extension as a whole, there is no easy way. With no function provided by `GL_ARB_arrays_of_arrays`, it's just simpler and easier to use a new capability for 2D array.